cpu/nrf52/Makefile.features

```diff
@@ -10,6 +10,9 @@ ifneq (,$(filter nrf52811xxaa nrf52820xxaa nrf52833xxaa nrf52840xxaa,$(CPU_MODEL
   FEATURES_PROVIDED += radio_nrf802154
 endif
 
+ifneq (,$(filter nrf5284%,$(CPU_MODEL)))
```

Comment:
AFAIK, only nrf52832 doesn't provide UARTE, so maybe just exclude this family ?
```suggestion
ifeq (,$(filter nrf52832%,$(CPU_MODEL)))
```

Reply:
It does not work on rnf51 either I believe.

Reply:
Yes, but this file is only for nrf52 :)

Reply:
it's really confusing that the nrf52832 is closer hardware wise to the nrf51 than the other members of the nrf52 family.
They should just have named it different.

Reply:
Anyway I prefer `ifneq` patterns than `ifeq` patterns if you don't mind. If there are other nrf52 susceptible to work I do not mind listing them.

Reply:
> Anyway I prefer ifneq patterns than ifeq patterns if you don't mind. If there are other nrf52 susceptible to work I do not mind listing them.

IMO, this is extra work each a new family is added, so subject to missing features unless someone remember to add it, and of course less maintainable.